client/src/components/Common/Map/index.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import styled from "styled-components";
+import GoogleMapReact from 'google-map-react'
+import { HiLocationMarker } from 'react-icons/hi';
+
+import { borderRadius, red1 } from '../../../resources/style-constants'
+
+const Container = styled.div`
+  width: 100%;
+  height: 30vw;
+  border-radius: ${borderRadius};
+  overflow: hidden;
+`;
+
+const MapContainer = styled.div`
+  height: 100%;
+  width: 100%;
+  border-radius: ${borderRadius};
+`;
+
+const MapPinContainer = styled.div`
+  width: 128px;
+  color: ${red1};
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+
+const MapPinIcon = styled.div`
+  height: 48px;
+  width: 48px;
+`;
+
+const MapPinLabel = styled.h4`
+  margin: 0;
+  padding: 0;
+  line-height: 0;
+`;
+
+const InvalidMapContainer = styled.div`
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+`;
+
+const MapPin = ({ text }) => {
```

Comment:
Since the MapPin is it's own isolated thing, you could place it in it's own folder/component that is nested inside of the Map folder.